com.github.wwmm.easyeffects.yml

```
@@ -58,14 +58,10 @@ modules:
   - name: easyeffects
     buildsystem: meson
     sources:
-      - type: archive
-        url: https://github.com/wwmm/easyeffects/archive/473cb6a985013f0e8af8136b53b27e64083ccc28.zip
-        sha256: cc8edcd8b760ee14411860e464d20ee0c96eb077d9760a9f24a9002b721f1283
-        x-checker-data:
-          type: html
-          url: https://raw.githubusercontent.com/vchernin/nightly-easyeffects-checker/main/latest_easyeffects_commit
-          version-pattern: (^.*$)
-          url-template: https://github.com/wwmm/easyeffects/archive/$version.zip
+      - type: git
+        url: https://github.com/wwmm/easyeffects.git
+        branch: master
+        commit: eb6937539ab21e76aa92e2cd6727468b7adcc72c
```

```suggestion
        branch: libadwaita
        commit: 473cb6a985013f0e8af8136b53b27e64083ccc28
```

Thanks for this!

Upstream development and some important bug fixes reside currently in `libadwaita`, so I wanted to have that in flathub-beta for now.

---

Oh, ok, as you say.